src/coreclr/hosts/coreshim/CoreShim.h

```diff
@@ -5,7 +5,6 @@
 #define _CORESHIM_H_
 
 // Platform
-#define NOMINMAX
```

Comment:
Is this now being inherited?

Reply:
Yes this is now specified at the command line so we don't need to do it in source.